packages/web/public/logos/coinbase.svg

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?><!-- Uploaded to: SVG Repo, www.svgrepo.com, Generator: SVG Repo Mixer Tools -->
+<svg width="800px" height="800px" viewBox="0 0 1024 1024" xmlns="http://www.w3.org/2000/svg">
+   <circle cx="512" cy="512" r="512" style="fill:#0052ff"/>
```

Comment:
This logo looks off.. where did you get it? Is there a way we can get an official logo?

Reply:
From here:  https://www.svgrepo.com/svg/331345/coinbase-v2

Reply:
I can switch it for this one:  https://logosarchive.com/wp-content/uploads/2021/12/Coinbase-icon-symbol-1.svg

Reply:
This one has the proper logo